### PR TITLE
[ERMP] task1: add 4 methods of component creating

### PR DESCRIPTION
**Pull request by arhigod**
image

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                }
+                <ReactComponent/>
+                <ReactPureComponent/>
+                {
+                    getFunctionalComponent()
+                }
+            </div>
+        );
+    }
+}
```
Comment (Pristavka) [inline src/components/App.js]: don't forget about empty line at the end of the files

File: src/index.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import { render } from 'react-dom';
 
+import App from './components/App';
+
 render(
     <div>
```
Comment (Pristavka) [inline src/index.js]: do we need this extra div?

File: src/components/App.js (hunk 1 above)
Comment (arhigod) [reply src/components/App.js]: Sorry, wrong beautifier config.
Fixed.

File: src/index.js (hunk 2 above)
Comment (arhigod) [reply src/index.js]: No :)

**Review by AnnZanevskaya — APPROVED** [review]

**Pull request closed without merging**